### DISCUSSION TITLE
Can awswrangler.s3.read_csv be used to read .txt files?

### DISCUSSION THREAD

**Discussion by ANicolucciTreeOfLife** [aws/aws-sdk-pandas#1742] [Q&A]
I have flat files that will be uploaded to AWS S3 as | delimited. can awswrangler.s3.read_csv method be used to read the files?
If not, what is the right method to read it with AWS Data Wrangler?

**Comment by malachi-constant** [comment]
Hi @ANicolucciTreeOfLife.

Possibly [`awswrangler.s3.read_fwf()`](https://aws-sdk-pandas.readthedocs.io/en/stable/stubs/awswrangler.s3.read_fwf.html#awswrangler.s3.read_fwf) would work for your use case? If there isn't fixed-width formatting in the text file, there aren't any other options in the library. 

Full list of s3 read methods are [listed here in the docs.](https://aws-sdk-pandas.readthedocs.io/en/stable/api.html#amazon-s3)

**Reply by ANicolucciTreeOfLife** [reply]
No, the files are not fixed-width formatting they are Pipe (|) delimited. I'm not sure why this method would be an option?

**Reply by malachi-constant** [reply]
Apologies, must have misread this. Theoretically this should work as we support passing any pandas supported kwargs in `read_csv()`. 

Have you tried this call with `sep='|'` yet by any chance? Let me know if that works and if not, I can open an issue to look into getting this supported.
